### ISSUE TITLE
Issue with last upload joomla-pkg_gantry5_v5.4.35

### ISSUE THREAD

**Issue by Johann009** [gantry/gantry5#2698]
Hi,
I use joomla 3.9.21 and Gantry5 with the Helium Rocket theme template.
after installing the latest version of the joomla plugin: joomla-pkg_gantry5_v5.4.35 - my contact form is no longer displayed. I looked at all the parameters, the form is no longer displayed. I reinstalled the version joomla-pkg_gantry5_v5.4.34 and the contact form is displayed correctly. 
So there is a problem with this last update.
Do you agree?
Thank you for your comment.

**Comment by simmonsr** [comment]
What contact form are you using?

**Comment by Johann009** [comment]
The native contact form of joomla.

I prepare a template page "contact" in Gantry with a header, an intro text, etc... 
I display the contact form from the main menu by choosing a contact form then I select my "contact" page template. 
The contact form is automatically integrated in the mainbar.
It has always worked very well. The issue appears with the last update! The information of the contact and the "send" button  appear BUT the fields to be completed are not displayed ...

**Comment by simmonsr** [comment]
Do you have user linked? I made that mistake too and it was not displaying:

![Screen Shot 2020-09-09 at 10 58 05 AM](https://user-images.githubusercontent.com/5002814/92623141-7ecd9680-f28b-11ea-8181-ab288d64d2c8.jpg)

**Comment by simmonsr** [comment]
Displays fine in 5.4.34 and 5.4.35.

**Comment by Johann009** [comment]
Okay, it works.
I downloaded this time joomla-pkg_gantry5_v5.4.35 from gantry.org and installed it manually. It works fine without anything else to do.
The first time I did this by going through the update manager without downloading it first. Does it make a difference?
Or was there an error/bug during the update?

Ok. Everything works fine.
Thanks for listening.
Sorry for the misunderstanding
kind regards
